worker/src/RTC/RtpPacket.cpp

```diff
@@ -597,6 +619,30 @@ namespace RTC
 		SetExtensionLength(this->midExtensionId, midLen);
 	}
 
+	void RtpPacket::UpdateDependencyDescriptor(const uint8_t* data, size_t len)
+	{
+		MS_TRACE();
+
+		uint8_t extenLen;
+		uint8_t* extenValue = GetExtension(this->dependencyDescriptorExtensionId, extenLen);
+
+		if (!extenValue)
+		{
+			MS_ERROR("Dependency description not found");
```

Comment:
please, no uppercase logs... 
Also, do we really need a MS_ERROR? It's not an error, it's a warning because it depends on what the client sends, right? we almost never use MS_ERROR

Reply:
Right.., leftovers from developement..